src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/AddAuthorGuestTypeFormSubscriber.php

```diff
@@ -40,11 +40,11 @@ public function preSetData(FormEvent $event)
         $form = $event->getForm();
         /** @var ReviewInterface $review */
         $review = $event->getData();
-        $author = $form->getConfig()->getOption('author');
+        $currentLoggedAuthor = $form->getConfig()->getOption('author');
```

Comment:
$currentlyLoggedInAuthor? That being said, I am not sure we should talk about being logged in here, because this can be any author that we want to specify.